components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/resources/dbScripts/h2.sql

```diff
@@ -804,20 +925,291 @@ CREATE TABLE IF NOT EXISTS IDN_CERTIFICATE (
             CONSTRAINT CERTIFICATE_UNIQUE_KEY UNIQUE (NAME, TENANT_ID)
 );
 
-CREATE TABLE IF NOT EXISTS IDN_OIDC_SCOPE (
-            ID INTEGER NOT NULL AUTO_INCREMENT,
-            NAME VARCHAR(255) NOT NULL,
-            TENANT_ID INTEGER DEFAULT -1,
-            PRIMARY KEY (ID)
-);
-
 CREATE TABLE IF NOT EXISTS IDN_OIDC_SCOPE_CLAIM_MAPPING (
             ID INTEGER NOT NULL AUTO_INCREMENT,
-            SCOPE_ID INTEGER,
-            EXTERNAL_CLAIM_ID INTEGER,
+            SCOPE_ID INTEGER NOT NULL,
+            EXTERNAL_CLAIM_ID INTEGER NOT NULL,
             PRIMARY KEY (ID),
-            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OIDC_SCOPE(ID) ON DELETE CASCADE,
-            FOREIGN KEY (EXTERNAL_CLAIM_ID) REFERENCES IDN_CLAIM(ID) ON DELETE CASCADE
+            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OAUTH2_SCOPE(SCOPE_ID) ON DELETE CASCADE,
+            FOREIGN KEY (EXTERNAL_CLAIM_ID) REFERENCES IDN_CLAIM(ID) ON DELETE CASCADE,
+            UNIQUE (SCOPE_ID, EXTERNAL_CLAIM_ID)
+);
+
+CREATE TABLE IF NOT EXISTS IDN_FUNCTION_LIBRARY (
+	NAME VARCHAR(255) NOT NULL,
+	DESCRIPTION VARCHAR(1023),
+	TYPE VARCHAR(255) NOT NULL,
+	TENANT_ID INTEGER NOT NULL,
+	DATA BLOB NOT NULL,
+	PRIMARY KEY (TENANT_ID,NAME)
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_CIBA_AUTH_CODE (
+    AUTH_CODE_KEY CHAR (36),
+    AUTH_REQ_ID CHAR (36),
+    ISSUED_TIME TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSUMER_KEY VARCHAR(255),
+    LAST_POLLED_TIME TIMESTAMP NOT NULL,
+    POLLING_INTERVAL INTEGER,
+    EXPIRES_IN  INTEGER,
+    AUTHENTICATED_USER_NAME VARCHAR(255),
+    USER_STORE_DOMAIN VARCHAR(100),
+    TENANT_ID INTEGER,
+    AUTH_REQ_STATUS VARCHAR (100) DEFAULT 'REQUESTED',
+    IDP_ID INTEGER,
+    UNIQUE(AUTH_REQ_ID),
+    PRIMARY KEY (AUTH_CODE_KEY),
+    FOREIGN KEY (CONSUMER_KEY) REFERENCES IDN_OAUTH_CONSUMER_APPS(CONSUMER_KEY) ON DELETE CASCADE
```

Comment:
Shall we use similar indentations?

Reply:
Check in other places too...

Reply:
Addressed with commit 930e08d24d16e5479553cf6decc257e2bc0f4bb3 in https://github.com/wso2/carbon-identity-framework/pull/3774 PR